server.R

```diff
@@ -389,8 +420,9 @@ lvmwd_baseline <- function(){
   colnames(bill_2014) <- c( paste("B", 1:num_tiers, sep=""),
                             paste("BR", 1:num_tiers, sep=""),
                             "baseline_variable_bill")
-  bill_2014 <- bill_2014 %>% mutate(baseline_bill=baseline_variable_bill + 30.21) 
-  
+  bill_2014 <- bill_2014 %>% mutate(baseline_bill=baseline_variable_bill + 30.21)
+  #adding baseline usage
+  bill_2014$baseline_usage <- bill_2014$B1 + bill_2014$B2 + bill_2014$B3 + bill_2014$B4 + bill_2014$B5
```

Comment:
Should probably be `select(matches("[B][0-9]")) %>% rowSums()` here also, and with the other lines like it in the file.